src/Telemetry.ts

```diff
@@ -3,52 +3,39 @@
 
 import crypto from 'crypto';
 import os from 'os';
-import {OutputChannel, window, workspace} from 'vscode';
-import TelemetryReporter from 'vscode-extension-telemetry';
+import TelemetryReporter, {
+  type TelemetryEventMeasurements,
+  type TelemetryEventProperties,
+} from '@vscode/extension-telemetry';
 import {Constants} from './Constants';
 import {Output, OutputLabel} from './Output';
 
-class ExtensionTelemetry {
-  private readonly output?: OutputChannel;
-
+export const Telemetry = new (class {
   private readonly reporter?: TelemetryReporter;
 
   private readonly defaultProperties: {[key: string]: any} = {};
 
   constructor() {
-    const isEnableTelemetry = workspace.getConfiguration('telemetry').get('enableTelemetry') || true;
-    const isTestRun = process.env.CODE_TEST || false;
-    // make testing easier.
     const extensionKey = process.env.AIKEY || Constants.extensionKey;
-    if (isEnableTelemetry) {
-      if (isTestRun) {
-        this.output = window.createOutputChannel(OutputLabel.telemetryClient);
-      } else {
-        try {
-          this.reporter = new TelemetryReporter(Constants.extensionName, Constants.extensionVersion, extensionKey);
-          // set default values for machine/session ids
-          this.defaultProperties['common.vscodemachineid'] = generateMachineId();
-          this.defaultProperties['common.vscodesessionid'] = generateSessionId();
-        } catch (error) {
-          Output.outputLine(OutputLabel.telemetryClient, `Initialize done with error: ${(error as Error).message}`);
-        }
-      }
+    try {
+      this.reporter = new TelemetryReporter(Constants.extensionName, Constants.extensionVersion, extensionKey);
+      // set default values for machine/session ids
+      this.defaultProperties['common.vscodemachineid'] = generateMachineId();
+      this.defaultProperties['common.vscodesessionid'] = generateSessionId();
```

Comment:
Any ideas why we include these two lines? AFAIU they are already provided by `@vscode/extension-telemetry`

https://github.com/microsoft/vscode-extension-telemetry/blob/04ff08da26bc437e87bfb398ac63be2685576066/src/common/baseTelemetryReporter.ts#L159-L160

Do you think is safe to remove this? @kevinbluer @xhulz @michaeljohnbennett